src/test/java/TestMain.java

```diff
@@ -0,0 +1,52 @@
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestMain {
+
+    @Test
+    public void shouldReturnTwentyThree() {
+        //when
+        final int GIVEN_FOR_TEST_PURPOSE = 10;
```

Comment:
Dlaczego wielkimi literami - to nie jest stała statyczna? Swoją drogą to powinna być sekcja `given`